Documentation/install/upgrade.rst

```diff
@@ -336,9 +336,23 @@ The following metrics have been renamed:
 New Metrics
 ~~~~~~~~~~~
 
+  * ``cilium_endpoint_regenerations_total`` counts of all endpoint regenerations that have completed, tagged by outcome.
+  * ``cilium_k8s_client_api_calls_total`` is number of API calls made to kube-apiserver labeled by host, method and return code.
   * ``cilium_kvstore_quorum_errors_total`` counts the number of kvstore quorum
     loss errors. The label ``error`` indicates the type of error.
 
+Deprecated Metrics/Labels
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+  * ``cilium_endpoint_regenerations`` will be removed. Please use ``cilium_endpoint_regenerations_total`` instead.
+  * ``cilium_k8s_client_api_calls_counter`` will be removed. Please use ``cilium_k8s_client_api_calls_total`` instead.
+  * ``cilium_identity_count`` will be removed. Please use ``cilium_identity`` instead.
+  * ``cilium_policy_count`` will be removed. Please use ``cilium_policy`` instead.
+  * ``cilium_policy_import_errors`` will be removed. Please use ``cilium_policy_import_errors_total`` instead.
+  * ``cilium_bpf_map_ops_total{mapName}`` will be removed. Please use ``cilium_bpf_map_ops_total{map_name}`` instead.
+  * ``cilium_nodes_all_events_received_total{eventType}`` will be removed. Please use
+    ``cilium_nodes_all_events_received_total{event_type}`` instead.
```

Comment:
please give an example, it was not clear at a first sight what was the difference between `eventType` and `event_type`